File: GXpert/GXpert.Web/Modules/Masters/State/StateForm.cs
using Serenity.ComponentModel;
using System;

namespace GXpert.Masters.Forms;

[FormScript("Masters.State")]
[BasedOnRow(typeof(StateRow), CheckNames = true)]
public class StateForm
{
    public string Title { get; set; }
    public string ShortName { get; set; }
}

File: GXpert/GXpert.Web/Modules/ServerTypes/Masters/StateForm.ts
import { StringEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface StateForm {
    Title: StringEditor;
    ShortName: StringEditor;
}

export class StateForm extends PrefixedContext {
    static readonly formKey = 'Masters.State';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!StateForm.init)  {
            StateForm.init = true;

            var w0 = StringEditor;

            initFormType(StateForm, [
                'Title', w0,
                'ShortName', w0
            ]);
        }
    }
}